core/src/epicli/data/common/ansible/playbooks/roles/preflight/tasks/apply.yml

```diff
@@ -61,6 +63,13 @@
           {{ (_defined | selectattr('config.enabled') | list) + _undefined }}
         _unsupported_enabled_extensions: >-
           {{ _enabled_extensions | selectattr('name', 'in', unsupported_postgres_extensions[_pg_arch]) | list }}
+
+    - name: Check the number of PostgreSQL hosts for Repmgr
+      when: postgresql.specification.extensions.replication.enabled | default(false)
+      assert:
+        that: groups.postgresql | length <= 2
```

Comment:
Why we need to limit this to 2? I think after postgres role refactoring you've done here it's quite easy to handle bigger clusters, isn't it?

Reply:
This was my assumption to do the same way as we did before.

Reply:
Created a new task for that #2346